### PR TITLE
Mot notebooks draft

### PR DESCRIPTION
**Pull request by karineip**
### Description
Added 00_webcam and 01_training_introduction notebooks for mot scenario, following similar format as for od scenario. The focus is on 01 notebook here, as the "Predict" section will carry over to the 00 notebook. 

- Notebooks follow Casey's TrackingLearner wrapper formats.
- I have added the evaluate() function, to be added to TrackingLearner class later, in the 01 notebook for review. 
- Will work on utils functions: convert_vott_MOTxywh() and convert_trackingbbox_video()
- Will work on TrackingLearner plot_training_losses() function
- Will import Casey's wrappers when ready
- Will work on faq.md and readme.md

### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging` and not `master`
- [ ] I will squash merge this PR into `staging`

### CODE REVIEW

**Comment by PatrickBue** [comment]
@Karineip Looking at this PR now. I guess you mainly want feedback for the 01 notebook, right? By the way, I see this PR has a lot of non-code files in it which should not go into Github repositories (since they track history and would otherwise get too large). Can you please remove them. Guess this will then also leave the 01 notebook and maybe some of the code you wrote.

File: scenarios/multi_object_tracking/01_training_introduction.ipynb [hunk 1]
```diff
@@ -0,0 +1,820 @@
+{
```
Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>Please sort the imports first by sections: python internal libraries (os, sys, time), external libraries (numpy, fastai), then utils_cv. And within sections sort alphabetically</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>I think this cell needs to be above the import statements, similar to e.g. object detection. We noticed some problems otherwise.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <ul><li>[minor] "greatly enhance its performance." might be a bit confusing, since without training the accuracy would be 0 of course.</li><li>[minor] When referring to code, put inside brackets such as this: <code>unzip()</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>Here and other places in the notebook: All input we need from the user (such as location of data, or parameter values) needs to be in the same call above as EPOCHS and be written in capital letters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <ul><li>For mvp this is good. For the final version, we'll want to change this to the same XML syntax used for object detection. The only extra information in your dataset is the re-id value.</li><li>Does FairMOT only support a single class? If so, we could build a dedicated "can" dataset using simply a subset of the images we have already with only cans being annotated. </li><li>As for "<code>FridgeObjects.train", </code>that is FairMOT specific and probably not something we want to expose to the user.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <ul><li>This needs more high-level information, e.g. why are we using this video for evaluation? That's because we can train on still-frame, but evaluation of e.g. ID swtiches needs video</li><li>The whole section should be moved towards the end of the notebook, after model training is done and evaluation starts</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <ul><li>[minor] All annotation instructions for video could be in the FAQ (including the 30fps). </li><li>Does VOTT output annotations in this format? <code>[image] [xmin] [y_min] [x_max] [y_max] [label]</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>Maybe try to simplify, not sure if we need to explain here again what the format of the annotation should be.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <ul><li>[minor] typo. Think you mean <code>TrackingDataset</code> rather than TorchvisionDataset (not that I put the TrackingDataset inside ` since it's code)</li><li>Here and other places: internal implementation details can be hidden from the user, e.g. "wrapper in a way that can recognized by the FairMOT repo code"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>This section is hard to read. Can we simplify and just say that we train the model on our dataset, starting with a model which was pre-trained on ImageNet (or pre-trained on pedestrian datasets). Details such as the Adam algorithm, etc, either should be at a later point (after the high-level intro) or actually fully removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>[minor] To keep the notebook short and easy to read, i'd not mention the defaults here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>Seems too verbose. Could just say that we plot the loss here, and in the FAQ (if we want to) explain what the loss is composed of (though I think just a reference to the paper is sufficient)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>Like other sections, the first sentence should be more of a high-level introduction, and not jump straight into details. Please see how much text you can copy from e.g. the detection 01 notebook or the image classification 01 notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>Those variables, since it's user input, should all be in the cell above where EPOCHS is specified, and in capital letters. Likely though we only want to highlight the 2-3 important parameters. All other parameters we'll set defaults in the utils_cv implementation and won't over-write (or even mention) here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (PatrickBue) [inline scenarios/multi_object_tracking/01_training_introduction.ipynb:1]: <p>Guess we can just re-use the parameters from above, rather than to re-define det_thres etc here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

**Comment by PatrickBue** [comment]
@ karineip Thanks for this PR. It's good to see the draft, and to add real code once it's ready. Left some comments, mainly high-level around keeping this notebook simple to understand and read, without over-loading the user with details.
